src/node_url.h

```diff
@@ -444,7 +444,6 @@ static inline void PercentDecode(const char* input,
   XX(URL_FLAGS_INVALID_PARSE_STATE, 0x04)                                     \
   XX(URL_FLAGS_TERMINATED, 0x08)                                              \
   XX(URL_FLAGS_SPECIAL, 0x10)                                                 \
```

Comment:
Likely worthwhile to decrement each of the remaining flags accordingly so that there's not a gap.